applications/nrf_desktop/doc/config_channel.rst

```diff
@@ -401,7 +401,7 @@ Listener configuration
 ======================
 
 The configuration channel listener is an application module that provides a set of options that are accessible through the configuration channel.
-For example, depending on listener, it can provide the CPI option from :ref:`nrf_desktop_motion` or the option for searching for new peer from :ref:`nrf_desktop_ble_bond`.
+For example, depending on the listener, it can provide the CPI option from :ref:`nrf_desktop_motion` or the option for searching for new peer from :ref:`nrf_desktop_ble_bond`.
```

Comment:
```suggestion
For example, depending on the listener, it can provide the CPI option from :ref:`nrf_desktop_motion` or the option for searching for a new peer from :ref:`nrf_desktop_ble_bond`.
```
Missing indefinite article.